src/vrm/material/MToonMaterial.ts

```diff
@@ -110,8 +112,9 @@ export class MToonMaterial extends THREE.ShaderMaterial {
   public mainTex_ST: THREE.Vector4 = new THREE.Vector4(0.0, 0.0, 1.0, 1.0); // _MainTex_ST
   public shadeTexture: THREE.Texture | null = null; // _ShadeTexture
   // public shadeTexture_ST: THREE.Vector4 = new THREE.Vector4(0.0, 0.0, 1.0, 1.0); // _ShadeTexture_ST (unused)
-  public bumpScale = 1.0; // _BumpScale
   public normalMap: THREE.Texture | null = null; // _BumpMap. again, THIS IS _BumpMap
+  public normalMapType: THREE.NormalMapTypes = THREE.TangentSpaceNormalMap; // Three.js requires this
+  public normalScale: THREE.Vector2 = new THREE.Vector2(1.0, 1.0); // _BumpScale, in Vector2
```

Comment:
Is the type omittable? (if so, we can do the same for `normalMapType` ) and others
```suggestion
  public normalScale = new THREE.Vector2(1.0, 1.0); // _BumpScale, in Vector2
```

Reply:
sounds good

Reply:
It's weird that we cannot lint this using `@typescript-eslint/no-inferrable-types`

Reply:
Addressed in https://github.com/pixiv/three-vrm/pull/253/commits/65b2ab1717c0a7d49312a175f65c25b694131b9b